src/traces/surface/convert.js

```diff
@@ -248,10 +248,6 @@ proto.update = function(data) {
         opacity: 1
     };
 
-    if('cmin' in data && 'cmax' in data) {
-        params.intensityBounds = [ data.cmin, data.cmax ];
```

Comment:
how does one specifies the `lo` / `hi` bounds computed [here](https://github.com/gl-vis/gl-surface3d/commit/b657c8ad64164a33def767ff456a19cdd3243632#diff-25ca11a30189a873f81fbb2edf9f470aR967) ?